translations/de.ts

```diff
@@ -705,35 +706,36 @@ Disabling chat history now will leave the encrypted history intact (but not usab
         <location filename="../src/chatlog/content/filetransferwidget.cpp" line="139"/>
         <source>Location not writable</source>
         <comment>Title of permissions popup</comment>
-        <translation>Ordner nicht beschreibbar</translation>
+        <translation>Kann in diesen Ordner nicht schreiben</translation>
     </message>
     <message>
         <location filename="../src/chatlog/content/filetransferwidget.cpp" line="140"/>
         <source>You do not have permission to write that location. Choose another, or cancel the save dialog.</source>
         <comment>text of permissions popup</comment>
-        <translation>Du besitzt nicht die Rechte um hier eine Datei zu speichern. Bitte wähle einen anderen Ordner oder breche die Aktion ab.</translation>
+        <translation>Du besitzt nicht die Rechte, um hier eine Datei zu speichern. Bitte wähle einen anderen Ordner oder brich die Aktion ab.</translation>
     </message>
     <message>
         <location filename="../src/chatlog/content/filetransferwidget.cpp" line="308"/>
         <source>paused</source>
         <comment>file transfer widget</comment>
-        <translation>Pausiert</translation>
+        <translation>Angehalten</translation>
```

Comment:
Ein Video und dergleichen wird auch angehalten. oder "Mach mal Pause". Somit is "angehalten" auf jeden Fall korrekt. "Pausiert" ist nicht gänzlich falsch, aber wird eher beim Wandern oder als Lehnübersetzung verwendet.